puppet/modules/jenkins_master/manifests/init.pp

```diff
@@ -0,0 +1,197 @@
+class jenkins_master {
+
+  jenkins::config { 'config_hash':
```

Comment:
Note that this class has no parameters (https://github.com/voxpupuli/puppet-jenkins/blob/master/manifests/config.pp#L3) and gets the hash by configuring it on the main class:

https://github.com/voxpupuli/puppet-jenkins/blob/master/manifests/config.pp#L13
https://github.com/voxpupuli/puppet-jenkins/blob/master/manifests/init.pp#L74